src/common/aggregate/aggregator.js

```diff
@@ -82,12 +84,36 @@ export class Aggregator extends SharedContext {
     var hasData = false
     for (var i = 0; i < types.length; i++) {
       type = types[i]
-      results[type] = toArray(this.aggregatedData[type])
+      results[type] = this.toArray(type, this.aggregatedData[type])
+
       if (results[type].length) hasData = true
       delete this.aggregatedData[type]
     }
     return hasData ? results : null
   }
+
+  toArray (type, obj) {
+    if (typeof obj !== 'object') return []
+
+    return mapOwn(obj, (key, value) => this.getValue(key, value, type))
```

Comment:
The `key` parameter is never used and I would like to continue removing uses of `mapOwn`

```suggestion
    return Object.values(obj).map(value => this.getValue(value, type))
```